featuretools/tests/synthesis/test_dfs_method.py

```diff
@@ -331,7 +323,7 @@ def test_accepts_pd_dateoffset_training_window(datetime_es):
     assert (feature_matrix.index == feature_matrix_2.index).all()
 
 
-def test_calls_progress_callback(entities, relationships):
+def test_calls_progress_callback(pd_entities, relationships):
```

Comment:
similarly we can split up this test into two parts so we can run the cluster test separately